cmake/projects/libxml2/hunter.cmake

```diff
@@ -21,19 +32,26 @@ hunter_add_version(
     958ae70baf186263a4bd801a81dd5d682aedd1db
 )
 
-# Drop dependencies
-hunter_cmake_args(
-	libxml2
-        CMAKE_ARGS
-            EXTRA_FLAGS=--without-python --without-lzma --without-zlib
-)
-
 hunter_configuration_types(libxml2 CONFIGURATION_TYPES Release)
-hunter_pick_scheme(DEFAULT url_sha1_autotools)
 hunter_cacheable(libxml2)
-hunter_download(
-    PACKAGE_NAME libxml2
-    PACKAGE_INTERNAL_DEPS_ID "1"
-    PACKAGE_UNRELOCATABLE_TEXT_FILES
-    "lib/pkgconfig/libxml-2.0.pc"
-)
+if (MSVC)
+    hunter_pick_scheme(DEFAULT url_sha1_libxml2_msvc)
+    hunter_download(
+        PACKAGE_NAME libxml2
+        PACKAGE_INTERNAL_DEPS_ID "2"
```

Comment:
I don't think having two different `PACKAGE_INTERNAL_DEPS_ID` is a good idea.

Something like this should work:
```cmake
set(_libxml_unrelocatable_text_files PACKAGE_UNRELOCATABLE_TEXT_FILES "lib/pkgconfig/libxml-2.0.pc")
if(MSVC)
  set(_libxml_unrelocatable_text_files "")
endif()
hunter_download(
    PACKAGE_NAME libxml2
    PACKAGE_INTERNAL_DEPS_ID "2"
    ${_libxml_unrelocatable_text_files}
)
```